A <%= @exception.class %> occurred in <%= @runner.class %> pid[<%= $$ %>]:
* Process : <%= $$ %>
* Server  : <%= `hostname -s`.chomp %>
<%= "* Last URL:  #{$last_url}" if $last_url %>
<%= "*      Log:  #{@log}" if @log %>

  <%= @exception.message %>
  <%= @backtrace.join "\n" %>